src/Page/Food/Builder.elm

```diff
@@ -773,11 +804,36 @@ packagingListView db selectedImpact recipe results =
     ]
 
 
+transportToDistributionView : Impact.Definition -> Recipe -> Recipe.Results -> List (Html Msg)
+transportToDistributionView selectedImpact recipe results =
+    [ span [ class "w-50" ] [ text "" ]
+    , div [ class "DownArrow" ]
+        [ text "" ]
+    , div
+        [ class "w-50"
+        , style "padding" ".5rem 1rem"
+        ]
+        [ div []
+            [ text "Masse : "
+            , Recipe.getTransformedIngredientsMass recipe
+                |> Format.kg
+            , text " + Emballage : "
+            , Recipe.getPackagingMass recipe
+                |> Format.kg
+            ]
```

Comment:
Have you tried moving this line to the left side of the down arrow?

Example:

```elm
transportToDistributionView : Impact.Definition -> Recipe -> Recipe.Results -> List (Html Msg)
transportToDistributionView selectedImpact recipe results =
    [ div [ class "w-50 fs-7 px-2 py-3 text-center" ]
        [ text "Masse : "
        , Recipe.getTransformedIngredientsMass recipe
            |> Format.kg
        , text " + Emballage : "
        , Recipe.getPackagingMass recipe
            |> Format.kg
        ]
    , div [ class "DownArrow" ] []
    , div [ class "w-50 fs-7 px-2 py-3" ]
        [ div [ class "d-flex justify-content-around" ]
            [ TransportView.entry results.distribution.transports.road Icon.bus "Transport routier"
            , Format.formatFoodSelectedImpact selectedImpact results.distribution.transports.impacts
            ]
        ]
    ]
```

![image](https://user-images.githubusercontent.com/41547/221637435-cc70c2b0-edda-4791-8d0f-3ff6222158f1.png)

Reply:
It looks nice indeed, however I've implemented as discussed and iterated with Pascal and Laurent (see the notion user story ;)